src/IssueViz/Editor/ErrorTagging/ErrorTagger.cs

```diff
@@ -45,9 +45,15 @@ protected override TagSpan<IErrorTag> CreateTagSpan(IIssueLocationTag trackedTag
         }
 
         protected override IEnumerable<IMappingTagSpan<IIssueLocationTag>> Filter(IEnumerable<IMappingTagSpan<IIssueLocationTag>> trackedTagSpans) =>
-            trackedTagSpans.Where(x => IsValidPrimaryLocation(x.Tag.Location));
+            trackedTagSpans.Where(x => IsValidPrimaryLocation(x.Tag.Location) && IsNotSuppressed(x.Tag.Location));
```

Comment:
You could write the checks as follows (which I think should give the same result):

```
        protected override IEnumerable<IMappingTagSpan<IIssueLocationTag>> Filter(IEnumerable<IMappingTagSpan<IIssueLocationTag>> trackedTagSpans) =>
            trackedTagSpans.Where(x => x.Tag.Location is IAnalysisIssueVisualization issueViz
                && !issueViz.IsSuppressed
                && IsValidPrimaryLocation(issueViz));

        private static bool IsValidPrimaryLocation(IAnalysisIssueVisualization viz) => viz.Span.IsNavigable();
```

There are a couple of advantages:
* the cast to `IAnalysisIssueVisualization` is only done once
* there is a minor performance gain in checking `IsSuppressed` first, since it is a simple property. It's only a small difference, but this method is called thousands to times so it will add up. 

I didn't add a helper method for `!issueViz.IsSuppressed` as I though it was simple enough not to need one.

Reply:
Ahh okay, yea the reason I wrote a helper method is because I was unaware of this syntax -> x.Tag.Location **is** 
so it was the only way for me to cast things but this is much better!